reference/behavior_test/establishing_decorator_pattern.rst

```diff
@@ -0,0 +1,369 @@
+Establishing Decorator Pattern
```

Comment:
Weird to use "establishing" here, what about "Our Decorator Pattern" or I don't get it